cfsdk/src/main/java/io/harness/cfsdk/cloud/analytics/AnalyticsPublisherService.java

```diff
@@ -15,48 +13,40 @@
 import io.harness.cfsdk.cloud.analytics.model.Metrics;
 import io.harness.cfsdk.cloud.analytics.model.MetricsData;
 import io.harness.cfsdk.cloud.core.client.ApiException;
-import io.harness.cfsdk.cloud.core.model.FeatureConfig;
-import io.harness.cfsdk.cloud.core.model.Variation;
-import io.harness.cfsdk.cloud.model.Target;
 import io.harness.cfsdk.logging.CfLog;
 
 /**
  * This class prepares the message body for metrics and posts it to the server
  */
 public class AnalyticsPublisherService {
 
-    private static final String FEATURE_NAME_ATTRIBUTE;
-    private static final String VARIATION_VALUE_ATTRIBUTE;
-    private static final String VARIATION_IDENTIFIER_ATTRIBUTE;
-    private static final String TARGET_ATTRIBUTE;
-    private static final Set<Target> globalTargetSet;
-    private static final Set<Target> stagingTargetSet;
-    private static final String JAR_VERSION;
+    private static final String SERVER;
     private static final String SDK_TYPE;
-    private static final String CLIENT;
-    private static final String SDK_LANGUAGE;
     private static final String SDK_VERSION;
+    private static final String SDK_LANGUAGE;
+    private static final String GLOBAL_TARGET;
+    private static final String TARGET_ATTRIBUTE;
+    private static final String FEATURE_NAME_ATTRIBUTE;
+    private static final String VARIATION_IDENTIFIER_ATTRIBUTE;
 
     static {
 
-        CLIENT = "client";
+        SERVER = "server";
```

Comment:
should this change to server, or should it still be client ?

Reply:
@davejohnston It is a client now. I have pushed the change.